### ISSUE TITLE
Method for obtaining the Alpha value with more decimals

### ISSUE THREAD

**Issue by jeronimoek** [elchininet/ColorTranslator#90]
Hi! I'm currently trying to implement this library in a visual studio code extension. It works pretty well, the only issue that I found is that when you parse a HEXA color, and then format it back to HEXA, due to the rounding to the Alpha value, #112233**50** turns into #112233**4F**

Example code:
```
const colorRaw = '#11223350'
const color = new ColorTranslator(colorRaw);
console.log(color.HEXA);                       // #1122334F
```

Relevant code:
```
// Public alpha property
public get A(): number {
  return round(this.hsl.a, 2);
}
```

I would like to know if you know any roadblocks in order to fix this. I'm willing to contribute if necessary

**Comment by elchininet** [comment]
Hi @jeronimoek,
Thanks for reaching out. I will take a look as soon as I can.
Of course, contributions are always welcomed!
Regards.

**Comment by jeronimoek** [comment]
Ok, nice! Later I'll give it a try.
Regards.

**Comment by elchininet** [comment]
Hi @jeronimoek,
The issue is solved, you can try the [version 1.10.2](https://www.npmjs.com/package/colortranslator/v/1.10.2) of the package.
Regards and thanks one more time for reaching out to fix an issue.
